package.json

```diff
@@ -84,5 +84,9 @@
       "android",
       "ios"
     ]
+  },
+  "config": {
+    "ionic_bundler": "webpack",
+    "ionic_source_map_type": "#inline-source-map"
```

Comment:
I confirmed that this works, but if I get that right this option will make bundle much bigger as source map is included into it: